app/scripts/views/mixins/open-gmail-mixin.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+
+/**
+ * A view mixin that helps confirm* views show the "Open Gmail" button
+ * directly from FxA.
+ */
+define(function (require, exports, module) {
+  'use strict';
+
+  return {
+    events: {
+      'click #open-gmail': '_gmailTabOpened'
+    },
+
+    /**
+     * Check if the `Open Gmail` button should be visible
+     *
+     * @param {string} email
+     * @returns {boolean}
+     */
+    isOpenGmailButtonVisible: function (email) {
+      // The "Open Gmail" is only visible in certain contexts
+      // we do not show it in mobile contexts because it performs worse on mobile
+      return this.broker.hasCapability('openGmailButtonVisible') &&
+             email.indexOf('@gmail.com') > 0;
```

Comment:
Would using a regular expression make sense here? We could test whether @gmail.com is at the end of string. `/(@gmail.com)$/`